lib/paratii.eth.vids.js

```diff
@@ -25,14 +43,16 @@ export class ParatiiEthVids {
     }
     options = dopts(options, defaults)
     let contract = await this.getRegistry()
-    let tx = await contract.methods.registerVideo(options.id, options.owner, options.price, options.ipfsHash).send()
+    contract.setProvider(this.eth.config.provider)
```

Comment:
if we need to call `contract.setProvider(..)`, we should probably do that in `this.getRegistry()`